scripts/quickstart.sh

```diff
@@ -101,7 +101,7 @@ conf() {
     # Extract SUPERBLOCKS_AGENT_CUSTOM_DOMAIN from SUPERBLOCKS_AGENT_HOST_URL
     if [ "$1" = "SUPERBLOCKS_AGENT_HOST_URL" ]; then
         if [[ $2 == http* ]]; then
```

Comment:
speaking of posix compliance, `[[ ]]` is not posix compliant either. If we don't expect users to run using bash, we should fix this everywhere

Reply:
looks like it's just this instance

Reply:
nice catch, fixing. and yeah, the shebang is set to `sh`, which is what im going by